app/src/main/res/values-sw600dp-port/dimens.xml

```diff
@@ -54,6 +54,7 @@
   <dimen name="previous_responses_item_exploration_view_margin_top">24dp</dimen>
   <dimen name="previous_responses_item_question_split_view_margin_top">24dp</dimen>
   <dimen name="previous_responses_item_question_view_margin_top">24dp</dimen>
+  <dimen name="previous_response_header_padding_top">24dp</dimen>
```

Comment:
As these values are not different from the ones in `values/dimens.xml` so they are not needed to be repeated here. Also we should remove the top_margins as they are already set to be space_0dp